vncviewer/Viewport.cxx

```diff
@@ -598,13 +600,40 @@ int Viewport::handle(int event)
   case FL_DRAG:
   case FL_MOVE:
   case FL_MOUSEWHEEL:
-    buttonMask = 0;
+
+    // FIXME: FLTK is not consistent in how it sends FL_PUSH/FL_RELEASE
+    // events:
+    //   * https://github.com/fltk/fltk/issues/1076
+    //   * https://github.com/fltk/fltk/issues/1077.
```

Comment:
This seems to be more about a missing FL_LEAVE. So a workaround for just the first three buttons seems insufficient?

Can't we work around this more easily by looking at the FL_UNFOCUS?

Reply:
Indeed, the issue still persists for the back/forward buttons. Resetting the state at `FL_UNFOCUS` would be a cleaner workaround.

Reply:
This is no longer an issue as FLTK is in the progress of adding support for back/forward in their API:

* https://github.com/fltk/fltk/pull/1081
* https://github.com/fltk/fltk/pull/1083